### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by MetaMmodern**
Update documentation due to changes in https://github.com/docker/getting-started/pull/104

### Proposed changes

I've added a second version of Dockerfile, because I had issues with building sql3 with yarn. 

### Related issues (optional)

https://github.com/docker/getting-started/pull/104
https://github.com/docker/getting-started/issues/36
https://github.com/docker/getting-started/issues/36

### CODE REVIEW

File: get-started/02_our_app.md [hunk 1]
```diff
@@ -49,7 +49,15 @@ see a few flaws in the Dockerfile below. But, don't worry! We'll go over them.
     RUN yarn install --production
     CMD ["node", "src/index.js"]
     ```
-
+    If you are going to use Docker on Windows, then the DOckerfile content should look like as following. (This is because For non-amd64 platforms we need python, make and g++ to compile sqlite3 node module)
+     ```dockerfile
+    RUN apk add --no-cache python g++ make
```
[the remark this answers is not in the document]
Comment (MetaMmodern) [reply get-started/02_our_app.md:54]: Hi) No problem, I will update the pr according to your notes. It is an honor for me to contribute to such a great project)

**Comment by MetaMmodern** [comment]
> Thanks @MetaMmodern for your contribution!
> You're right, this line needs to be added to this description as well.
> I left a comment to just show one Dockerfile instead of two. Do you want to update your PR?

Sure, I'll update it asap)

**Comment by usha-mandya** [comment]
Hey @MetaMmodern, wondering if you had a chance to update the docs? Thanks again for your contribution. Appreciate it :)

**Comment by MetaMmodern** [comment]
> Hey @MetaMmodern, wondering if you had a chance to update the docs? Thanks again for your contribution. Appreciate it :)

Sorry, totally forgot about it) gonna do this right now)

**Comment by MetaMmodern** [comment]
@usha-mandya Hi, I've finished fixing my fix))))

File: get-started/02_our_app.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MetaMmodern) [reply get-started/02_our_app.md:47]: Done)

**Review by StefanScherer — APPROVED** [review]
LGTM

**Pull request merged**